File: JS/custo.js
const assert = require('assert');

const custoFinal = (cFabrica) => {
  let percentD = 1.28;
  let percentI = 1.45;
  let custoFinal = cFabrica + (cFabrica * percentI - cFabrica) + (cFabrica * percentD - cFabrica); 
  return custoFinal;
}

try {
  assert.equal(17300,custoFinal(10000));
} catch (e) {
  console.log(e);
}

File: C/custo.c
#include <stdio.h>
#include <assert.h>

float custoFinal(float custoF) 
{
  float custoFim = custoF + (custoF * 0.45f) + (custoF * 0.28f);
  return custoFim;
}

int main()
{
  assert(17300.0f == custoFinal(10000));
  return 0;
}